.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
.vscode should be ignored by git. Not sure why it got pushed up. Please remove it and update the gitignore